source/stylesheets/modules/_toc.scss

```diff
@@ -44,3 +44,7 @@
     display: none;
   }
 }
+
+.toc-link--in-view {
```

Comment:
Yeah, it is weird. I don't think it's too much of an issue to send the extra HTML over the wire, personally.

I think modifying the TOC generator is out of the scope of this PR, but perhaps we should change this to something like

```scss
.toc-link {
  &--in-view {
    font-weight: bold;
  }
}
```

...just to make it look a bit less weird?